app/assets/javascripts/angular/controller/home-controller.js.erb

```diff
@@ -5,9 +5,24 @@ app.controller('HomeCtrl', ['$scope', '$http', '$rootScope', function($scope, $h
     author: "author (you)",
     article_text: "Put a birthday message here"
   };
-  $scope.card = {
-    date: new Date()
-  }
+
+  
+
+  $scope.clear = function() {
+    $scope.card.date = null;
+  };
+  $scope.minDate = new Date("1946-01-01");
+  $scope.maxDate = new Date();
+  $scope.open = function($event) {
+    $event.preventDefault();
+    $event.stopPropagation();
```

Comment:
yes you are right. I'm not using it - left over from playing with the datepicker